app/components/articleSearch/index.tsx

```diff
@@ -15,6 +15,7 @@ import selectPapers from "./select";
 import { trackAndOpenLink } from "../../helpers/handleGA";
 import { ICurrentUserRecord } from "../../model/currentUser";
 import checkAuthDialog from "../../helpers/checkAuthDialog";
+import { parse } from "query-string";
```

Comment:
Can u change this library to [qs](https://github.com/ljharb/qs)?
Because this library can't block the malicious script injection attack.